components/ContactForm.js

```diff
@@ -0,0 +1,18 @@
+var contactForm = React.createClass({
```

Comment:
class name should start with big letter - you're using class ContactForm with first big caps